Xcode/SDL/pkg-support/resources/CMake/sdl3-config.cmake

```diff
@@ -39,7 +39,7 @@ string(REGEX REPLACE "SDL3\\.framework.*" "" SDL3_FRAMEWORK_PARENT_PATH "${CMAKE
 set_and_check(SDL3_PREFIX       "${SDL3_FRAMEWORK_PATH}")
 set_and_check(SDL3_EXEC_PREFIX  "${SDL3_FRAMEWORK_PATH}")
 set_and_check(SDL3_INCLUDE_DIR  "${SDL3_FRAMEWORK_PATH}/Headers")
-set(SDL3_INCLUDE_DIRS           "${SDL3_INCLUDE_DIR};${SDL3_FRAMEWORK_PATH}")
```

Comment:
The above might mean this is wrong. I'm not sure how CMake and frameworks interact, but I can run some tests for you here if that's helpful.

Reply:
`SDL3_FRAMEWORK_PATH` is the path of the framework: `/path/to/SDL3.framework`.
So the include path should be the `Headers` subfolder. In that folder, there should be a `SDL3` subfolder containing the headers.

In any case, it would be interesting to do a pre-pre-release of SDL3 when the time is ready to check your release scripts and whether I didn't screw up anything.